docs/product-user-guide/src/main/asciidoc/interacting-with-processes-delegating-claiming-releasing-tasks-proc.adoc

```diff
@@ -23,7 +23,7 @@ Once tasks are created in {CENTRAL}, you can delegate them to others, claim rele
 
 .Procedure
 . Go to *Menu* -> *Manage* -> *Tasks*.
```

Comment:
I see that you have switched from Task Inbox to Tasks perspective in case of claiming the task procedure.
A task can be claimed from Task Inbox page as well, in case the task is assigned to a group which the user belongs to. Such tasks would have the status **Ready** (i.e. ready for claiming).
Maybe it's worth changing to Task Inbox, for the sake of consistency and not introducing the confusion around possible restrictions on claiming tasks.

Reply:
Change ack.
Change to Task Inbox.